src/api.js

```diff
@@ -1,5 +1,4 @@
 /* @flow */
-'use strict';
```

Comment:
The best option I have seen so far is to keep the "use strict" and add ` // eslint-disable-line strict` so that it doesn't create an error.